src/Ecal/EcalVetoProcessor.cxx

```diff
@@ -650,6 +895,42 @@ std::vector<std::pair<float, float> > EcalVetoProcessor::getTrajectory(
   return positions;
 }
 
+// MIP tracking functions:
+
+/**
+ * Returns the distance between the lines v and w, with v defined to pass through
+ * the points (v1,v2) (and similarly for w).
+ * 
+ * @param[in] v1 An arbitrary point on line v
+ * @param[in] v2 A second, distinct point on line v
+ * @param[in] w1 An arbitrary point on line w
+ * @param[in] w2 A second, distinct point on line w
+ * @returns Closest distance of approach of lines u and v
+*/
+float EcalVetoProcessor::distTwoLines(TVector3 v1, TVector3 v2, TVector3 w1, TVector3 w2) {
+  TVector3 e1 = v1 - v2;
+  TVector3 e2 = w1 - w2;
+  TVector3 crs = e1.Cross(e2);
+  if (crs.Mag() == 0) {
+    return 100.0;  // arbitrary large number; edge case that shouldn't cause
+                   // problems.
+  } else {
+    return std::abs(crs.Dot(v1 - w1) / crs.Mag());
+  }
+}
+
+/**
+ * Return the minimum distance between the point h1 and the line passing through points p1 and p2.
+ *
+ * @param[in] h1 Point to find the distance to
+ * @param[in] p1 An arbitrary point on the line
+ * @param[in] p2 A second, distinct point on the line
+ * @returns Minimum distance between h1 and the line
+ */
```

Comment:
Sorry for not being clear. This documentation should be above the function _declaration_.

For example, these lines should be above this line: 
```
  float distTwoLines(TVector3 v1, TVector3 v2, TVector3 w1, TVector3 w2);
```
In EcalVetoProcessor.h

Reply:
Thanks for clarifying--just fixed it.